proxy/server/target.go

```diff
@@ -512,6 +563,49 @@ func convertStatus(s *status.Status) *pb.Status {
 	}
 }
 
+// An unconnected client stream is a grpc.ClientStream
```

Comment:
This feels more useful than just here in proxy. Perhaps one of the util packages?

Reply:
per-offline chat. We'll wait until we have another potential callsite before moving this to a utility library.